Examples/Modules/dive_cleaning/analysis.py

```diff
@@ -15,13 +15,16 @@
 to the theoretical field of a Gaussian beam.
 """
 import sys
+
 import matplotlib
+
 matplotlib.use('Agg')
 import matplotlib.pyplot as plt
-import yt
 import numpy as np
 import scipy.constants as scc
```

Comment:
The scipy imports could be kept together.

Reply:
yeah, that's curious

Reply:
haha, it really does not like it.

Ah no, it's clear: the first block is `import ...` statements, then next follow the `from ...` statements

Reply:
@dpgrote does this look better? I added two options in `.editorconfig` :)